docs/local_setup_with_docker.md

```diff
@@ -4,7 +4,7 @@ These instructions assume you have Git, Homebrew, Postgres, Ruby, and Docker ins
 
 ## Setting up the environment
 
-Let's start by setting up our environment. If you haven't already, create a `.env` file and add the following (populating with your own preferences, where applicable):
+Let's start by setting up our environment. If you haven't already, create a `.env.local` file and add the following (populating with your own preferences, where applicable):
```

Comment:
similar thought to above, i think it'd be better to get away from hard-coding the required variables in the README as well as in the example env files